requirements.txt

```diff
@@ -2,6 +2,5 @@ tox>=2.5.0,<3.0.0
 nose==1.3.7
 mock==1.3.0
 wheel==0.24.0
-docutils>=0.10,<0.16
```

Comment:
I think we should remove it from the `requirements-docs.txt` as well. There is not really a point to keep this around since the file is really only used to help with HTML doc generation and `docutils` was never actually ever used to generate botocore/boto3 docs.

Reply:
We actually need this here because older versions of Sphinx have a relatively unbounded dependency on docutils. We'll need to install it until we upgrade Sphinx, so this identifier let's us keep control for now. I'll make a TODO to work on the Sphinx changes in a future PR.